config/initializers/geocoder.rb

```diff
@@ -2,6 +2,7 @@
   # geocoding options
   # :timeout      => 3,           # geocoding service timeout (secs)
   # :lookup       => :google,     # name of geocoding service (symbol)
+  :lookup => :test,
```

Comment:
I don't think so, wasn't sure of the best way to check this, tried making some checkins using Insomnia using different lat/lngs and I was getting the correct addresses back rather than just Denham. Is there a better way to check? I'm probably being stupid.